src/Stryker.Core/Stryker.Core.UnitTest/Mutators/MathMutatorTest.cs

```diff
@@ -0,0 +1,146 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Shouldly;
+using Stryker.Core.Mutators;
+using Xunit;
+
+namespace Stryker.Core.UnitTest.Mutators
+{
+    public class MathMutatorTest : TestBase
```

Comment:
I would like to see a test to see if these methods are *not* mutated when the class has a different name. I can imagine we will see a class with a method Ceiling and we could mutate that as well (which will fail)

Reply:
What about `using static System.Math` or `using AnyName = System.Math`?

Reply:
I guess you should handle `using static System.Math` which implies supporting the short call form `Cos(x)` : easy to do and with limited compilation errors risk (those should only happens WITHIN a class providing methods with mathematical names).

I think the alias form is so rare there is no need to support it. Remember: mutation testing is about stress testing your tests, this is not some form of extensive testing.

It should be easy to properly support aliasing when the decision is made to generate semantic information alongside the syntax tree (if ever).

Reply:
> I think the alias form is so rare there is no need to support it.

Murphy's law says that whatever can happen will happen :D